src/vs/vscode.proposed.d.ts

```diff
@@ -1368,6 +1361,7 @@ declare module 'vscode' {
 		language: string;
 		outputs?: NotebookCellOutput[];
 		metadata?: NotebookCellMetadata;
+		previousResult?: NotebookCellPreviousExecutionResult;
```

Comment:
I'm assuming these properties are read-write.
Asking as there' no ctor that accepts this value.
@roblourens /cc

Reply:
Should be in the constructor. The `previousResult` stuff needs a little work here, I swear I lost some changes in a stash somewhere...

Reply:
at least you stashed, i never pop my stashes, always apply (for this reason, been burned far too many times)...